### ISSUE TITLE
Stuck PRs

### ISSUE THREAD

**Issue by ironashram** [maas/terraform-provider-maas#97]
Hi, 
i am opening this issue hoping to speed things up a bit, there are a couple of very much needed changes that have not been accepted yet.

At the moment i am maintaining a fork of this provider https://github.com/maas/terraform-provider-maas/compare/master...ironashram:terraform-provider-maas:merge_features?expand=1

Basically i have included the following PRs and a few small adjustments here and there:

- https://github.com/maas/terraform-provider-maas/pull/77
- https://github.com/maas/terraform-provider-maas/pull/7
- Replaced a deprecated package
- Added a couple of tests for the new maas_machine data source and some other minor fixes

Should i open a new PR with all of the above? Separate ones? or can we somehow unblock the existing ones?

Thank you

**Comment by troyanov** [comment]
Hello @ironashram 

I am sorry that MAAS Terraform provider felt off the radar for such a long time. However, now we have a bit more time to spend on it and as always, we are happy for external contributions.

Regarding these two PRs you've mentioned, I see that they require some changes. 
We will try to reach authors and if they won't respond in a timely manner, we will do those changes ourselves, since they both have toggled the option when "Maintainers are allowed to edit this pull request"

Regarding deprecated package update and more tests - we are more than happy to see PRs for this!

**Comment by jadonn** [comment]
If it makes any difference, interface resources for bonds, bridges, and VLANs would be extremely helpful for me as well. For example, I'm currently in the middle of writing Terraform plans to use the MAAS provider and the Juju provider to streamline deploying an OpenStack cloud, and provisioning an OVS bridge through the MAAS provider is one bit of functionality I'm missing that would be really useful in particular. I hope these features can be implemented soon! If I can do anything to help with moving them along, I'm happy to do so

**Comment by skatsaounis** [comment]
I am closing this since the only remaining #77 is revamped and ready to be reviewed,tested,merged,released. Thank you @ironashram for your patience and for bringing awareness. I hope that both of you will enjoy the new functionality as soon as we will release it :slightly_smiling_face:
